src/pilot/assign_role.py

```diff
@@ -861,9 +862,12 @@ def place_node_in_available_state(ironic_client, node_uuid):
 
 
 def assign_role(ip_mac_service_tag, node_uuid, role_index, os_volume_size_gb,
-                ironic_client, drac_client):
+                ironic_client, node_definition, node):
     flavor = ROLES[role_index.role]
-
+    drac_ip, drac_user, drac_password = \
```

Comment:
I don't understand why drac_client is being rebuilt here.  Can't it just be passed in by the caller?

Reply:
Modified this code 
Please refer https://github.com/dsp-jetpack/JetPack/pull/202